model/src/main/java/org/openrewrite/gradle/toolingapi/Assertions.java

```diff
@@ -115,11 +115,15 @@ public static UncheckedConsumer<List<SourceFile>> withToolingApi(@Nullable Gradl
                                 OpenRewriteModel model = OpenRewriteModelBuilder.forProjectDirectory(tempDirectory.resolve(sourceFile.getSourcePath()).getParent().toFile(), null, initScriptContents);
                                 org.openrewrite.gradle.toolingapi.GradleSettings rawSettings = model.gradleSettings();
                                 if (rawSettings != null) {
-                                    GradleSettings gradleSettings = org.openrewrite.gradle.toolingapi.GradleSettings. toMarker(rawSettings);
+                                    GradleSettings gradleSettings = org.openrewrite.gradle.toolingapi.GradleSettings.toMarker(rawSettings);
                                     sourceFiles.set(i, sourceFile.withMarkers(sourceFile.getMarkers().add(gradleSettings)));
                                 }
                             } else {
-                                OpenRewriteModel model = OpenRewriteModelBuilder.forProjectDirectory(projectDir.toFile(), tempDirectory.resolve(sourceFile.getSourcePath()).toFile(), initScriptContents);
+                                File maybeBuildFile = null;
+                                if (sourceFile.getSourcePath().endsWith("build.gradle")) {
+                                    maybeBuildFile = tempDirectory.resolve(sourceFile.getSourcePath()).toFile();
+                                }
```

Comment:
build.gradle is the default, but they can be renamed. 
This will also break the `GradleProject` marker getting attached to build.gradle.kts files, which allows dependency search recipes to work on those even though we parse them only as plain text

Reply:
hmm good point. I also understood that basically all .gradle files, except .settings are build files and should probably be marked as such. I think with the approach we're planning this is no longer needed anyway. So closing this PR.